src/web/mjs/connectors/oremangaNet.mjs

```diff
@@ -0,0 +1,12 @@
+import oremangaNet from './templates/oremangaNet.mjs';
+
+export default class oremanga extends oremangaNet {
```

Comment:
Are you trying to extends a class that doesnt exists? 

Moreover class name 'oremanga' doesnt match filename 'oremangaNet' (case and name)